src/components/ReportActionItem/TaskView.js

```diff
@@ -53,6 +54,7 @@ function TaskView(props) {
     const canModifyTask = Task.canModifyTask(props.report, props.currentUserPersonalDetails.accountID);
     const disableState = !canModifyTask;
     const isDisableInteractive = !canModifyTask || !isOpen;
+    const personalDetails = usePersonalDetails() || CONST.EMPTY_OBJECT;
```

Comment:
Why doesn't `usePersonalDetails` return an empty object itself? Then we can get rid of all these `|| CONST.EMPTY_OBJECT`. Is the idea to make it more explicit?

Reply:
I would prefer not too. Let's keep it how it is in the PR, thanks!